tests/integration/connections/debug_envgetter.py

```diff
@@ -0,0 +1,27 @@
+import os
+import json
+import logging
+
+from databricks.labs.remorph.connections.env_getter import EnvGetter
+
+
+class TestEnvGetter(EnvGetter):
+    def __init__(self, is_debug: bool = True):
+        self.is_debug = is_debug
+        super().__init__()
```

Comment:
call to super() in __init__ is better coming first

Reply:
Done